arkouda/pdarrayclass.py

```diff
@@ -1355,6 +1357,10 @@ def std(pda : 'pdarray', ddof : int=0) -> np.float64:
     the estimated variance, so even with ``ddof=1``, it will not be an
     unbiased estimate of the standard deviation per se.
     """
+    if not isinstance(ddof):
```

Comment:
`ddof` is an instance of what?

Reply:
int, will fix, man, sorry about that. three errors already